ct/zigbee2mqtt.sh

```diff
@@ -32,91 +32,35 @@ function update_script() {
     msg_error "No ${APP} Installation Found!"
     exit
   fi
-  if [[ "$(node -v | cut -d 'v' -f 2)" == "18."* ]]; then
-    if ! command -v npm >/dev/null 2>&1; then
-      echo "Installing NPM..."
-      apt-get install -y npm >/dev/null 2>&1
-      echo "Installed NPM..."
-    fi
+  RELEASE=$(curl -s https://api.github.com/repos/Koenkk/zigbee2mqtt/releases/latest | grep "tag_name" | awk '{print substr($2, 2, length($2)-3) }')
+  if [[ ! -f /opt/${APP}_version.txt ]] || [[ "${RELEASE}" != "$(cat /opt/${APP}_version.txt)" ]]; then
+    msg_info "Stopping Service"
+    systemctl stop zigbee2mqtt
+    msg_ok "Stopped Service"
+
+    msg_info "Creating Backup"
+    mkdir -p /opt/z2m_backup
+    tar -czf /opt/z2m_backup/${APP}_backup_$(date +%Y%m%d%H%M%S).tar.gz -C /opt zigbee2mqtt &>/dev/null
+    mv /opt/zigbee2mqtt/data /opt/z2m_backup
+    msg_ok "Backup Created"
+
+    msg_info "Updating ${APP} to v${RELEASE}"
+    cd /opt
+    wget -q "https://github.com/Koenkk/zigbee2mqtt/archive/refs/tags/${RELEASE}.zip"
+    unzip -q ${RELEASE}.zip
+    mv zigbee2mqtt-${RELEASE} /opt/zigbee2mqtt
+    rm -rf /opt/zigbee2mqtt/data
+    mv /opt/z2m_backup/data /opt/zigbee2mqtt
+    cd /opt/zigbee2mqtt 
+    pnpm build &>/dev/null
+    pnpm install --frozen-lockfile &>/dev/null
```

Comment:
```suggestion
    pnpm install --frozen-lockfile &>/dev/null
    pnpm build &>/dev/null
```

These should be the other way around. Dependencies are used in the build process.